lib/librato/version_specifier.rb

```
@@ -0,0 +1,15 @@
+module VersionSpecifier
+  def self.supported(opts={})
+    return unless block_given?
+    return if !opts.key?(:min) && !opts.key?(:max)
```

prefer raising an exception to silent failure here if this is a requirement